happi/client.py

```diff
@@ -600,16 +603,24 @@ def _store(self, device, insert=False):
         post.update({'type': tpe,
                      'creation': creation,
                      'last_edit': ttime.ctime()})
+
         # Find id
         try:
-            _id = post[self._id_key]
+            # In case we want to update the name of an entry
+            # We want to add a new entry, and delete the old one
+            if the_old_name != post[self._id_key]:
+                _id = the_old_name
```

Comment:
The returned `_id` should be the valid one, not the returned one.

Reply:
Ok that makes sense, thank you - `the_old_name` won't be valid after this anymore... 
Fixed it in b69e45d - basically i am just returning `post[self._id_key]` ?? please let me know if that's not what you meant

Reply:
Another typo in my comment, apologies for the distracted commenting. I meant "not the ~returned~ deleted one"

That's correct 👍